stable/postgresql/templates/configmap.yaml

```diff
@@ -1,16 +1,13 @@
+{{ if  (.Files.Glob "files/postgresql.conf") }}
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: {{ template "postgresql.fullname" . }}
+  name: {{ template "postgresql.fullname" . }}-configuration
   labels:
     app: {{ template "postgresql.name" . }}
     chart: {{ template "postgresql.chart" . }}
-    release: {{ .Release.Name }}
-    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name | quote }}
+    heritage: {{ .Release.Service | quote }}
 data:
-  {{- if .Values.metrics.customMetrics }}
-  custom-metrics.yaml: {{ toYaml .Values.metrics.customMetrics | quote }}
-  {{- end }}
-  {{- if .Values.pgHbaConf }}
-  pg_hba.conf: {{ .Values.pgHbaConf | quote }}
```

Comment:
Do you intend to keep support somehow of pg_hba.conf?

Reply:
Hi! At this moment the chart supports custom configuration using `postgresql.conf`, but I am working to provide support for both (`postgresql.conf` and `pg_hba.conf`). But this change not only requires modifications in the chart (this is the easy part), it also requires changes in the Docker image.

TL;DR I am working to modify the Docker image to support it.

Do you think it is a blocking issue to merge it or can we merge the PR and send a new one with the `ph_hba.conf` changes after some days?

Reply:
It is blocking for me (I have a dozen of postgres running using that feature) for the long term but I guess it can wait for a few days with no harm and in case of problem I can use my own fork.

Reply:
Thanks!
I just added some logic to support it at chart level 92498ba, feel free to review it!
In the other hand I continue working on the Docker image stuff